build/README.md

```diff
@@ -0,0 +1,19 @@
+# SIG Doc builder
+
+This script will generate the following documentation files:
+
+```
+sig-*/README.md
+sig_list.md
+```
+
+Based off the `sigs.yaml` metadata file.
+
+## How to use 
+
+To (re)build documentation for all the SIGs, run these commands:
+
+```bash
+docker build -t sigdocs -c build/Dockerfile build
+docker run -v $(pwd):/go/src/app sigdocs
```

Comment:
My understanding was that the `golang:1.6-onbuild` image already does this. From [the docs](https://hub.docker.com/_/golang/):

> This image includes multiple ONBUILD triggers which should cover most applications. The build will COPY . /go/src/app, RUN go get -d -v, and RUN go install -v. This image also includes the CMD ["app"] instruction which is the default command when running the image without arguments.